### DISCUSSION TITLE
Mouse wheel VFO scroll problem

### DISCUSSION THREAD

**Discussion by m0euk** [AlexandreRouma/SDRPlusPlus#1522] [General]
Apologies if this is covered elswehere, couldn't find it. Really enjoying the software on an M3 Mac Mini running an Airspy HF+ Discovery. Works very well indeed. One minor issue i've got is erratic behaviour from the mouse wheel when trying to scroll up and down frequencies. The expected behaviour of a single notch on the wheel sometimes has the VFO move in the corresponding direction, and sometimes does nothing. Scrolling more quickly jumps up or down faster as expected, but isn't predictable or consistent in how far it goes. By comparison the same mouse in other software e.g. SDRConnect, or non-radio software like Google Sheets etc, scrolls as you would expect. Its only a minor thing, and I'm using the keyboard left and right arrows to step up and down frequencies instead, but thought I'd flag it. 73.

**Comment by craigerl** [comment]
I made a quick patch for this issue, if it helps,

https://github.com/AlexandreRouma/SDRPlusPlus/issues/1486
